README.md

```diff
@@ -2,12 +2,6 @@
 
 Thorn is composed of a set of helper abstractions built to ease the process of setting up a [SugarCRM](https://www.sugarcrm.com) instance's REST API testing environment and interacting with it.
 
-## Installation
```

Comment:
TBH, it makes sense to put it back. I think I was considering being more explicit about how Sugar uses Thorn internally when I originally took this out.